lib/linguist/languages.yml

```diff
@@ -1110,6 +1110,9 @@ Dockerfile:
   - ".dockerfile"
   filenames:
   - Dockerfile
+  - "dockerfile"
+  - "Dockerfile.*"
+  - "dockerfile.*"
```

Comment:
That's not going to work. You'll have to give the explicit name.

Reply:
That's precisely the issue @kmehant has raised in #4566: Linguist has no way of identifying files based on a common *prefix*, only a *suffix*. Although that gives me an idea...